box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestsFile.java

```diff
@@ -1159,4 +1193,52 @@ protected void onSendCompleted(BoxResponse<BoxVoid> response) throws BoxExceptio
             super.handleUpdateCache(response);
         }
     }
+
+    public static class DownloadRepresentation extends BoxRequestDownload<BoxDownload, DownloadRepresentation> {
+
+        /**
+         * String sufix to be used when fetching a representation page
+         */
+        private static final String REPRESENTATION_PAGE_ASSET_EXTENSION = ".png";
```

Comment:
Technically this can also be jpg right? Maybe we shouldn't make this a static final, but instead a member variable that defaults to png.